packages/chat-headless/tests/chatheadless.test.ts

```diff
@@ -28,7 +29,13 @@ const mockedMetaState: MetaState = {
 };
 
 beforeEach(() => {
-  jest.spyOn(coreLib, "provideChatCore").mockImplementation();
+  jest.spyOn(coreLib, "provideChatCore").mockImplementation(() => {
+    const client: ChatHttpClient = {
```

Comment:
is this change necessary?

Reply:
Yeah, since `resetConversation` now uses `isChatEventClient`, `provideChatCore` can no longer return `undefined`.